runtime/tests/integration_tests.rs

```diff
@@ -132,6 +140,57 @@ fn test_fw_info() {
     assert_eq!(info.pl0_pauser, 0xFFFF0000);
 }
 
+#[test]
+fn test_invoke_dpe_get_profile_cmd() {
+    let mut model = run_rom_test("mbox");
```

Comment:
I think we should make this an RT test: `run_rt_test(None)`. A lot of the DPE functionality depends on things being set up by ROM/FMC (like the DICE keys)